src/components/shared/flyout/flyoutSection.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Btn,
+  SectionHeader,
+  SectionDesc
+} from 'components/shared';
+import { svgs, joinClasses } from 'utilities';
+
+import './styles/flyoutSection.css';
+
+export class FlyoutSection extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showContent: true
+    };
+
+    this.toggleContent = this.toggleContent.bind(this);
+  }
+
+  toggleContent() {
+    this.setState({ showContent: !this.state.showContent });
+  }
+
+  render() {
```

Comment:
It would be better to implement a provider pattern here so the local accordion state is captured in the component context. Then we can keep the nice declarative approach of making sections.

Reply:
I am not sure what you mean in this context.